src/beach.ts

```diff
@@ -0,0 +1,730 @@
+  { minute: 8994, row: 10, column: 5 },
+  { minute: 9000, row: 9, column: 9 },
+  { minute: 9036, row: 2, column: 3 },
+  { minute: 9053, row: 5, column: 3 },
+  { minute: 9059, row: 2, column: 8 },
+  { minute: 9118, row: 7, column: 2 },
+  { minute: 9120, row: 6, column: 0 },
+  { minute: 9125, row: 3, column: 4 },
+  { minute: 9139, row: 9, column: 8 },
+  { minute: 9150, row: 10, column: 3 },
+  { minute: 9156, row: 10, column: 1 },
+  { minute: 9172, row: 7, column: 9 },
+  { minute: 9194, row: 9, column: 2 },
+  { minute: 9211, row: 9, column: 0 },
+  { minute: 9216, row: 6, column: 8 },
+  { minute: 9238, row: 7, column: 3 },
+  { minute: 9244, row: 8, column: 5 },
+  { minute: 9245, row: 4, column: 9 },
+  { minute: 9254, row: 7, column: 4 },
+  { minute: 9265, row: 8, column: 6 },
+  { minute: 9281, row: 9, column: 0 },
+  { minute: 9290, row: 8, column: 6 },
+  { minute: 9293, row: 7, column: 2 },
+  { minute: 9304, row: 2, column: 5 },
+  { minute: 9343, row: 8, column: 8 },
+  { minute: 9347, row: 3, column: 4 },
+  { minute: 9364, row: 4, column: 4 },
+  { minute: 9375, row: 10, column: 1 },
+  { minute: 9382, row: 7, column: 7 },
+  { minute: 9405, row: 9, column: 2 },
+  { minute: 9415, row: 10, column: 2 },
+  { minute: 9445, row: 10, column: 1 },
+  { minute: 9453, row: 7, column: 1 },
+  { minute: 9460, row: 4, column: 9 },
+  { minute: 9470, row: 8, column: 2 },
+  { minute: 9472, row: 8, column: 1 },
+  { minute: 9473, row: 3, column: 4 },
+  { minute: 9493, row: 7, column: 3 },
+  { minute: 9515, row: 5, column: 0 },
+  { minute: 9520, row: 9, column: 2 },
+  { minute: 9523, row: 5, column: 5 },
+  { minute: 9525, row: 8, column: 9 },
+  { minute: 9538, row: 8, column: 8 },
+  { minute: 9562, row: 3, column: 5 },
+  { minute: 9602, row: 5, column: 2 },
+  { minute: 9605, row: 8, column: 2 },
+  { minute: 9616, row: 8, column: 9 },
+  { minute: 9634, row: 1, column: 6 },
+  { minute: 9652, row: 8, column: 0 },
+  { minute: 9666, row: 7, column: 0 },
+  { minute: 9684, row: 9, column: 5 },
+  { minute: 9693, row: 7, column: 7 },
+  { minute: 9698, row: 9, column: 2 },
+  { minute: 9702, row: 5, column: 7 },
+  { minute: 9713, row: 3, column: 7 },
+  { minute: 9725, row: 3, column: 0 },
+  { minute: 9728, row: 4, column: 6 },
+  { minute: 9758, row: 5, column: 3 },
+  { minute: 9812, row: 3, column: 0 },
+  { minute: 9830, row: 5, column: 2 },
+  { minute: 9867, row: 8, column: 2 },
+  { minute: 9877, row: 7, column: 7 },
+  { minute: 9877, row: 8, column: 0 },
+  { minute: 9891, row: 9, column: 9 },
+  { minute: 9913, row: 10, column: 5 },
+  { minute: 9966, row: 8, column: 4 },
+  { minute: 9971, row: 8, column: 2 },
+  { minute: 9971, row: 10, column: 3 },
+  { minute: 9976, row: 8, column: 7 },
+  { minute: 9992, row: 9, column: 2 },
+];
+
+function _comb(tile: BeachTile): void {
+  const { minute, row, column } = tile;
+  cliExecute(`beach wander ${minute}; beach comb ${row} ${column}`);
```

Comment:
What do we want to do if the tide is up and we can't hit a given tile? One option is to just omit all of those tiles from the pool.

Reply:
Very good point! The short answer is "I don't know how the beach comb works well enough to have planned for this."

Would love to know the long answer. If we skip them, then the subsequent tiles have a higher load.

Reply:
the kol beach, as with any normal beach, has tides. at low tide, the entire 10x10 grid is accessible. each day in the beach cycle, the tide changes -- it goes up one row for 4 days and down one row for 4 days. so the bottom 4 rows are not always accessible, especially the bottom-most one, which is only available once every eight days. the tides are reset at rollover, while the rest of the beach is not, which makes it even weirder in terms of accessibility.

Reply:
Maybe we should comment some of this information into the code?

Reply:
I think I will probably port at least some of this into libram. Definitely the functions for tidal prediction.